docs/UserGuide.md

```diff
@@ -1,42 +1,150 @@
-# User Guide
+# Patient Manager User Guide
 
-## Introduction
+Patient Manager is a _Command Line Interface_ (CLI) application for _general practitioners_ (GP) 
+who work in clinics to manage their patient list. This includes a recording/retrieval of 
+past record of visit, scheduling of the next appointment, and some other features listed below. 
+With the Patient Manager, GPs will be able to reduce paperwork and have a more efficient way 
+to organize the records of their patients.
 
-{Give a product intro}
+---
 
-## Quick Start
+## Table of Contents
 
-{Give steps to get started quickly}
+* [Quickstart](#quickstart)
+* [Features](#features)
+    * [Print a help message: `help`](#print-a-help-message-help)
+    * [Adding a patient: `add`](#adding-a-patient-add)
+    * [Listing all patients: `list`](#listing-all-patients-list)
+    * [Loading a patient's medical records: `load`](#loading-a-patients-medical-records-load)
+    * [Displaying the current loaded patient: `current`](#displaying-the-current-loaded-patient-current)
+    * [Recording a patient's consultation details: `record`](#recording-a-patients-consultation-details-record)
+    * [Retrieving a patient's consultation details: `retrieve`](#retrieving-a-patients-consultation-details-retrieve)
+    * [Exiting the program: `exit`](#exiting-the-program-exit)
+* [Command summary](#command-summary)
 
-1. Ensure that you have Java 11 or above installed.
-1. Down the latest version of `Duke` from [here](http://link.to/duke).
+---
 
-## Features 
+## Quickstart
 
-{Give detailed description of each feature}
+1. Ensure that you have Java 11 installed on your local computer. If you do not have the correct version
+   of Java installed, `Java 11` can be downloaded from 
+   [here](https://docs.aws.amazon.com/corretto/latest/corretto-11-ug/downloads-list.html).
+   
+1. Obtain a copy of the latest version of the Patient Manager (tp.jar) from 
+   [here](https://github.com/AY2021S2-CS2113T-W09-4/tp/releases) and place it in an empty folder.
+   
+1. Open a terminal/command line (cmd)/powershell. A Windows 10 OS' screenshot is here:\
+   \
+   ![PowerShell](./images/WindowsPowerShell.png)
+   
+1. Execute `java -jar tp.jar` to start the Patient Manager.
 
-### Adding a todo: `todo`
-Adds a new item to the list of todo items.
+1. Once the welcome message appears, simply type in a command (e.g. [`help`](#print-a-help-message-help)) and hit `ENTER` at the end.
```

Comment:
Probably a renumbering is required (1. 2. 3. 4. ...)?

Reply:
Initially I had them labelled 1. 2. 3. but according to the [markdown coding standard](https://se-education.org/guides/conventions/markdown.html), using generic numbers (1.) makes it easier to insert more items in the future.

Reply:
Oh. I notice that. I will merge it then. Thanks!